public/lessons/javascript/the_command_line.md

```diff
@@ -139,17 +145,21 @@ mkdir code
 * `mkdir` ("make dir") -- creates a new SUBDIRECTORY inside the current directory
 * `cd` ("change dir") -- move into a different directory
 
-> These apply to Mac & Unix shells, but most work in Windows too
+> These apply to Mac & Unix shells, Windows equivalents are below.
+* `cd` ("change dir") -- With no directory, it lists the current directory. Otherwise, it changes to the specified directory
+* `dir` ("directory") -- shows the contents of the current directory
+* `mkdir` ("make dir") `` creates a new SUBDIRECTORY inside the current directory
+
 
 # LAB: make a subdirectory and then enter it
 
-1. open Terminal
+1. open **Terminal** or **Command Prompt**
 2. make a new subdirectory using `mkdir code`
 3. change into that directory using `cd code`
 4. make sure you're really there using `pwd`
-4. list its contents using `ls` (and note that it's empty)
-  * **On Windows** you might need to type `dir` instead of `ls`
-
+   * **On Windows** use `cd`
```

Comment:
So, if you do a plain `cd` in cmd.exe does it show you the current directory? In unix, that changes to the home dir.